streamerate/tests/test_Streams.py

```diff
@@ -1428,6 +1482,12 @@ def test_to_list(self):
         s = stream(range(3))
         self.assertListEqual(s.to_list(), [0, 1, 2])
 
+    def test_to_dataframe(self):
+        self.skipTest(
+            "Not implemented, to avoid making Pandas a dependency of "
```

Comment:
This is okay to implement, as the Pandas will be a dependency just of the tests, not of the library, so feel free to include the pandas as dependency to the `pyproject.toml` -> `[tool.poetry.dev-dependencies]`